fastlane/Fastfile

```diff
@@ -76,6 +76,9 @@ GH_ORG_NAME="woocommerce"
 
     ios_bump_version_release()
     new_version = ios_get_app_version()
+    extract_release_notes_for_version(version: new_version,
+      release_notes_file_path: FIle.join(ENV["PROJECT_ROOT_FOLDER"], 'RELEASE-NOTES.txt'),
+      extracted_notes_file_path: './WooCommerce/Resources/release_notes.txt')
```

Comment:
Any reason for not using `File.join` and `PROJECT_ROOT_FOLDER` here?

Reply:
Yes, the reason is because I copy/pasted this path from another line down below in the same Fastfile verbatim, and didn't think about converting it to `File.join` while I was at it 😅

Good point, `File.join` it should be.